Flutter/cartesian-charts/chart-types/spline-area-chart.md

```diff
@@ -116,13 +150,15 @@ The following code sample demonstrates how to set the `splineType` value to `car
 
 {% endhighlight %}
 
-![Spline area type](cartesian-chart-types-images/spline_area_types.png)
+![Dashed spline area chart](cartesian-chart-types-images/spline_area_dashed.png)
+
+
 
-## See Also
+#### See Also
 
-* [Color palette](./series-customization#color-palette) 
-* [Color mapping](./series-customization#color-mapping-for-data-points)
-* [Animation](./series-customization#animation)
-* [Gradient](./series-customization#gradient-fill)
-* [Empty points](./series-customization#empty-points)  
-* [Sorting](./series-customization##sorting) 
\ No newline at end of file
+* [color palette](https://help.syncfusion.com/flutter/cartesian-charts/series-customization#color-palette) 
```

Comment:
I didn't asked to change the link like this, the existing itself fine, why you have changed like this?

Reply:
Hi @dharanidharandharmasivam ,

existing link is not open properly, only the above type only opens, so I have changed.

regards,
@PavithraNandhiniRamachandran